src/adapter/src/coord/sequencer.rs

```diff
@@ -1053,46 +1056,64 @@ impl Coordinator {
 
         self.catalog_transact(Some(session), vec![op]).await?;
 
-        self.create_cluster_replica(cluster_id, id).await;
+        self.create_cluster_replicas(&[(cluster_id, id)]).await;
 
         Ok(ExecuteResponse::CreatedClusterReplica)
     }
 
-    async fn create_cluster_replica(&mut self, cluster_id: ClusterId, replica_id: ReplicaId) {
-        let cluster = self.catalog.get_cluster(cluster_id);
-        let replica_config = cluster.replicas_by_id[&replica_id].config.clone();
+    async fn create_cluster_replicas(&mut self, replicas: &[(ClusterId, ReplicaId)]) {
+        let mut log_source_ids: HashMap<_, Vec<_>> = HashMap::new();
+        let mut log_source_collections_to_create = vec![];
```

Comment:
Nit, for consistency:

```suggestion
        let mut log_source_collections_to_create = Vec::new();
```

Reply:
omg, i prefer `Vec::new()` too, idk what i was doing here...